src/lib/radio/radio.ts

```diff
@@ -396,6 +406,9 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
   /** Whether the ripple effect on click should be disabled. */
   private _disableRipple: boolean;
 
+  /** Current color. Can be `primary`, `accent`, or `warn`. */
+  private _color: string;
```

Comment:
The type should use the string literal values
```ts
private _color: 'primary' | 'accent' | 'warn';
```

Reply:
Is there any way to use straight color values like for example `#00FF00` or `rgb(255,100,0)` from HTML ?